gym-crowdedness.html

```diff
@@ -0,0 +1,383 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="UTF-8">
+<title>Gym Crowdedness - iodide</title>
+<link rel="stylesheet" type="text/css" href="https://iodide-project.github.io/master/iodide.master.css">
+</head>
+<body>
+<script id="jsmd" type="text/jsmd">
+%% meta
+{
+  "title": "Gym Crowdedness",
+  "lastSaved": "2018-03-27T03:08:12.605Z",
+  "lastExport": "2018-03-27T03:37:06.899Z"
+}
+
+%% md
+# Exploring the Campus Gym Crowdedness Dataset
+
+## Purpose
+The purpose of this notebook is to explore and analyze the Campus Gym Crowdedness Dataset for correlations between the number of people in the campus gym and other features such as temperature and time. 
+
+## Technologies Used
+- D3.js
+- plotly.js
+
+## Data
+Each row in the dataset represents how many people were in the campus gym at a given time. Measurements were taken every 10 minutes at the campus gym over the course of a year. Other feature columns such as temperature and day of week have been added to the dataset.
+
+Source:
```

Comment:
This should be near the very top of the notebook, near where you'll put your author name :)